### ISSUE TITLE
template with lookup failing if selecting the organism

### ISSUE THREAD

**Issue by sergiocontrino** [intermine/intermine#978]
to replicate:
import template
```xml
<template name="Search_Term_Protein_FASTA" title="Gene --&gt; Protein FASTA" comment="">
  <query name="Search_Term_Protein_FASTA" model="genomic" view="Protein.genes.primaryIdentifier Protein.primaryIdentifier Protein.genes.length Protein.sequence.length Protein.sequence.residues" longDescription=" " sortOrder="Protein.primaryIdentifier asc">
    <constraint path="Protein.genes" editable="true" op="LOOKUP" value="ARRB*" extraValue=""/>
  </query>
</template>
```
it works with 'any' for the organism, while it fails with 

Ooops...! Could not load the data-table.
Service failed. Please contact support.

when an organism is selected from the drop-down list.

stack trace:
```java
ERROR org.intermine.webservice.server.WebService     - Service failed by internal error. Request parameters: 
query: <query model="genomic" view="Protein.id" name="Search_Term_Protein_FASTA" ><constraint path="Protein.genes" op="LOOKUP" value="ARR*" extraValue="A. thaliana" code="A" /><constraint path="Protein.genes.id" op="IS NOT NULL" /></query>
format: jsoncount
token: c144CcBbN835Sbc8u5z7
java.lang.IllegalArgumentException: Query 
SELECT DISTINCT a1_.id AS a2_, a1_.primaryIdentifier AS a3_, a1_.secondaryIdentifier AS a4_, a1_.symbol AS a5_ FROM org.intermine.model.bio.Gene AS a1_, org.intermine.model.bio.Organism AS a6_ WHERE ((LOWER(a1_.primaryIdentifier), LOWER(a1_.secondaryIdentifier), LOWER(a1_.symbol)) IN ? AND a1_.organism CONTAINS a6_ AND a6_.shortName = 'A. thaliana') 1: [] does not contain any BagConstraints
	at org.intermine.api.bag.BagQuery.getQueryForWildcards(BagQuery.java:168)
	at org.intermine.api.bag.BagQueryRunner.search(BagQueryRunner.java:194)
	at org.intermine.api.bag.BagQueryRunner.searchForBag(BagQueryRunner.java:90)
	at org.intermine.api.query.MainHelper.putConstraintsInMap(MainHelper.java:668)
	at org.intermine.api.query.MainHelper.makeQuery(MainHelper.java:357)
	at org.intermine.api.query.MainHelper.makeQuery(MainHelper.java:139)
	at org.intermine.api.query.WebResultsExecutor.makeQuery(WebResultsExecutor.java:165)
	at org.intermine.api.query.WebResultsExecutor.makeQuery(WebResultsExecutor.java:146)
	at org.intermine.api.query.WebResultsExecutor.explain(WebResultsExecutor.java:119)
	at org.intermine.web.struts.CreateTemplateAction.execute(CreateTemplateAction.java:132)
```

**Comment by julie-sullivan** [comment]
1. unrelated to templates
2. happens in beta not live flymine so a new bug, introduced by me no doubt.
3. Error is on the LOOKUP on Protein.genes, so it's not handling the paths correctly.

**Comment by julie-sullivan** [comment]
Fixed by 3896038ddac497e753a77aaf8f2e5f3d33661b61
